User-Data Store Migration/README.md

```diff
@@ -0,0 +1,85 @@
+# User-Data Store Migration Tool
+
+## Context
+Linkurious Enterprise is shipped with an embedded Sqlite database
+to allow an immediate usage of the system in a PoC environment, however it's a best practice to relay on a relational databased management system (RDBMS) in a production environment.
```

Comment:
avoid contracted forms (`it's`, `we're`) in formal documents, prefer regular forms: ` it is`, `we are`, etc.